index.html

```diff
@@ -1,54 +1,378 @@
+              <a href="#"><button class="button-3">Ruby</button></a>
+            </div>
+          </div>
+          <div class="button-box">
+            <a href="#"><button class="button-4">See Project</button></a>
+          </div>
+        </div>
+
+        <!-- card No. 5 -->
+
+        <div class="cards">
+          <div class="text-box">
+            <div>
+              <h3 class="text-box-heading">Profesional Art Printing Data</h3>
+            </div>
+            <div>
+              <p class="text-box-paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+            </div>
+            <div class="text-box-buttons">
+              <a href="#"><button class="button-3">html</button></a>
+              <a href="#"><button class="button-3">bootstrap</button></a>
+              <a href="#"><button class="button-3">Ruby</button></a>
+            </div>
+          </div>
+          <div class="button-box">
+            <a href="#"><button class="button-4">See Project</button></a>
+          </div>
+        </div>
+
+        <!-- card No. 6 -->
+
+        <div class="cards">
+          <div class="text-box">
+            <div>
+              <h3 class="text-box-heading">Profesional Art Printing Data</h3>
+            </div>
+            <div>
+              <p class="text-box-paragraph">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+            </div>
+            <div class="text-box-buttons">
+              <a href="#"><button class="button-3">html</button></a>
+              <a href="#"><button class="button-3">bootstrap</button></a>
+              <a href="#"><button class="button-3">Ruby</button></a>
+            </div>
+          </div>
+          <div class="button-box">
+            <a href="#"><button class="button-4">See Project</button></a>
+          </div>
+        </div>
+      </section>
+
+      <!-- About me section start from here -->
+
+      <section class="about-me">
+        <!-- About-me container start from here -->
+
+        <div class="about-me-container">
+          <!-- About-me heading start from here -->
+
+          <div class="about-me-heading">
+            <h1>About me</h1>
+          </div>
+          <div class="about-me-paragraph">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div>
+            <a href="#"><button class="button-2">Get My Resume</button></a>
+          </div>
+        </div>
+
```

Comment:
This is Figma Design:
![image](https://user-images.githubusercontent.com/61361037/212376542-01eac533-0fe8-4fc9-b960-0ee473f74284.png)

While this is Yours:
![image](https://user-images.githubusercontent.com/61361037/212376711-35257957-ad1d-4df2-b143-7106e05c780a.png)

Your Title and Description of About Me sections are pushed a little bit down. Kindly push it a little bit above and try to match Figma design as much as you can.